tests/cpp/test_multidevice_lower_communication.cpp

```diff
@@ -26,14 +26,18 @@ namespace {
 void assertIsCompiledToHostIrContainer(
     const FusionExecutorCache& executor_cache) {
   FusionKernelRuntime* runtime = executor_cache.getMostRecentKernelRuntime();
-  EXPECT_EQ(runtime->executors().size(), 1);
-  EXPECT_THAT(
-      runtime->executors(),
-      Each(Pointer(Property(
-          "is a HostIrExecutor",
-          &ExecutorAbstract::isA<HostIrExecutor>,
```

Comment:
This was intended to verify the fusion is compiled to communications. Recall that in the non host IR code path, HostIrExecutor is for and only for communications. 

Therefore, a better check would be to verify the number of Communication Exprs in the HostIrEvaluator's top-level expression list. In gmock, you can do this with `Contains(e).Times(n)`. See https://github.com/NVIDIA/Fuser/blob/6ff60e2a320733a2f49de57007d6bb45000107cd/tests/cpp/test_move_split_cat.cpp#L545 for an example.